app/services/injection_response_service.rb

```diff
@@ -10,11 +10,11 @@ def run!
     slack.build_injection_payload(@response)
     return failure(action: 'run!', uuid: @response['uuid']) unless @claim
 
-    injection_attempt = create_injection_attempt
+    injection_attempt
     if Settings.aws&.sqs&.response_queue_url
       slack.send_message!
```

Comment:
This if block was temp code for debugging the K8s injection.
It can, and should, be removed once that has become more stable and this will revert to the single line 17